Tribler/Core/Modules/restapi/upgrader_endpoint.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import absolute_import
+
+import logging
+
+from twisted.web import resource
+
+import Tribler.Core.Utilities.json_util as json
+
+
+class UpgraderEndpoint(resource.Resource):
+    """
+    With this endpoint you can control DB upgrade process of Tribler.
+    """
+
+    def __init__(self, session):
+        resource.Resource.__init__(self)
+        self._logger = logging.getLogger(self.__class__.__name__)
+        self.session = session
+
+    def render_DELETE(self, _):
```

Comment:
The semantics of this HTTP verb (`DELETE`) does not match the actual behaviour since it is not actually deleting the upgrader itself. I would construct an endpoint to skip the upgrade process as something like: `POST /upgrader/skip`.

Reply:
Fine point.